README.rst

```diff
@@ -1,7 +1,7 @@
 Ramos
 =====
 
-.. image:: https://travis-ci.org/luizalabs/ramos.svg?branch=master
+.. image:: https://travis-ci.org/luizalabs/ramos.svg
```

Comment:
if a pr fails on CI what status will show here?

Reply:
I will revert this.